packages/connector/src/index.ts

```diff
@@ -23,9 +23,7 @@ const defaultSnapOrigin = 'npm:@blockchain-lab-um/masca';
  * Install and enable Masca
  *
  * Checks for existence of MetaMask Flask and installs Masca if not installed
```

Comment:
Maybe add a more detailed description as this is probably the most important function for developers

Reply:
Added a note explaining only MM Flask should be enabled